encrypt/encrypt.go

```diff
@@ -73,6 +73,11 @@ func Encrypt(args []string) error {
 			outFilename = path.Join(*outDir, basename) + ".c4gh"
 		}
 
+		// check that the encrypted file already exists
+		if helpers.FileExists(outFilename) {
+			return fmt.Errorf("Encrypted filename %s already exists", outFilename)
+		}
+
```

Comment:
This fix does not seem to resolve the issue. 
For example, the output of  `./sda-cli encrypt -key c4gh.pub.pem dummy_data.c4gh` is `dummy_data.c4gh.c4gh`. Instead of checking if the file exists, it would be preferable to check if the input file is `*.c4gh`. Maybe even better, if the first 8 bytes of the input file are `crypt4gh`.

Reply:
Right, I will change it.

Reply:
Also, you could add a test about it in the respective tests file

Reply:
Yes! On it!